Files/Views/ModernShellPage.xaml

```diff
@@ -217,6 +217,7 @@
         <Frame
             x:Name="ItemDisplayFrame"
             Grid.Row="2"
+			CacheSize="1"
```

Comment:
Ignore the cache changes for now. Just a test I had going locally.

Reply:
@gave92 Before we approve and merge this, what are your thoughts on limiting the CacheSize?

Reply:
I think it's very good. I was afraid it would hurt navigation speed but it looks like it only has an effect when changing layout mode (as long as you navigate to the same page type the page's still cached).